test/core-test/quantum_objects_evo.jl

```diff
@@ -81,7 +81,7 @@
         @test adjoint(a2).data == adjoint(a2.data)
 
         N = 10
-        a = QobjEvo(destroy(N), Operator, N)
+        a = QobjEvo(MatrixOperator(sprand(ComplexF64, N, N, 5 / N)), Operator, N)
```

Comment:
Why do we need to construct `MatrixOperator` byourselves?
I thought we can use `QobjEvo(destroy(N), Operator, N)` directly

Reply:
Exactly, it was just to improve codecov.